### DISCUSSION TITLE
blog/the_second_high_school/introduction

### DISCUSSION THREAD

**Discussion by giscus** [yyyytawa/yyyytawa.github.io#14] [Announcements]
# blog/the_second_high_school/introduction

有一天,然后就想着做出了这个 命名规范: 遵循以下几点: 以大小类为基础的分类 如高一 高二 高三 高一下面有上学期,寒假,下学期,暑假等等 寒,暑假按照放假时所在的年级来算,如高一结束放的暑假放到高一的目录里面 原则上/学期按照第x周来记录,记录开学时间 (按到校时间算) 假期的事件按照发生时间来算 附件表: 班主任演讲次数/频率附件表

https://docs.yyyyt.top/blog/the_second_high_school/introduction.html

**Comment by runoneall** [comment]
几岁，未成年饮酒不可以！！！

**Reply by runoneall** [reply]
（（（

**Reply by yyyytawa** [reply]
114514岁
